project-management/zenhub.md

```diff
@@ -69,14 +69,26 @@ If you're waiting for something/ someone to continue work on an issue, move it t
 
 If you need to move on to something with a higher priority move any issue you're assigned but not working on back to `Current Iteration`.
 
-### Blocked
-
-Issues that are blocked for any reason. All blocked issues should have a comment explaining why they're blocked.
-
 ### Review/ QA
 
 Any issue that is deemed code/ UX/ design complete but requires a review or QA.
 
 ### Closed
 
 Anything that is complete. The final resting place for issues and epics.
+
+## Labels
```

Comment:
We made this label description [file](https://github.com/canonical-webteam/practices/blob/master/workflow/labels.md) to describe labels